### PR TITLE
Added Russian translation to UI_RU.txt

### PR DESCRIPTION
**Pull request by MRSEEO**
Added an amateur translation of the mod into Russian.
Please first take the file in English, change the encoding to Windows-1251 and then add my changes, otherwise the language will not work correctly.

### CODE REVIEW

File: Contents/mods/zomboid-cnd-speech/media/lua/shared/Translate/RU/UI_RU.txt [hunk 1]
```diff
@@ -32,297 +32,299 @@
...
-UI_Phrases_SWEAR4 = "shit",
-UI_Phrases_SWEAR5 = "fuck",
+UI_Phrases_SWEAR1 = "дерьмо",
+UI_Phrases_SWEAR2 = "чёрт",
+UI_Phrases_SWEAR3 = "чёрт побери",
+UI_Phrases_SWEAR4 = "пиздец",
+UI_Phrases_SWEAR5 = "блять",
+UI_Phrases_SWEAR6 = "сука",
+UI_Phrases_SWEAR7 = "ахуеть",
 --UI_Phrases_SWEARskipwords
```
Comment (ChuckTheSheep) [inline Contents/mods/zomboid-cnd-speech/media/lua/shared/Translate/RU/UI_RU.txt:328]: SWEARskipwords is used in interlacedFucks filter. Basically, "fuck" won't be placed before these words, and if any "fucks" are added after one of these words it is turned into "fucking" instead. I assume something like this doesn't really apply in Russian?

File: Contents/mods/zomboid-cnd-speech/media/lua/shared/Translate/RU/UI_RU.txt [hunk 2]
```diff
@@ -344,8 +346,8 @@ UI_Phrases_SWEARskipwords17 = "a",
 UI_Phrases_SWEARskipwords18 = "on",
 UI_Phrases_SWEARskipwords19 = "some",
 --UI_Phrases_FUCKS
-UI_Phrases_FUCKS1 = "fuck",
-UI_Phrases_FUCKS2 = "fucking",
+UI_Phrases_FUCKS1 = "ебать",
+UI_Phrases_FUCKS2 = "ёбанный",
 --UI_Phrases_Plosives
```
Comment (ChuckTheSheep) [inline Contents/mods/zomboid-cnd-speech/media/lua/shared/Translate/RU/UI_RU.txt:351]: Plosives are common letters that people may stutter/stammer on. Is there any such letters in Russian?

**Review by ChuckTheSheep — COMMENTED** [review]
Thank you so much for providing a translation!

File: Contents/mods/zomboid-cnd-speech/media/lua/shared/Translate/RU/UI_RU.txt (hunk 2 above)
Comment (MRSEEO) [reply Contents/mods/zomboid-cnd-speech/media/lua/shared/Translate/RU/UI_RU.txt:351]: Sure. You can stutter on any letters from the Russian language, except "й", "ь" и "ъ".

File: Contents/mods/zomboid-cnd-speech/media/lua/shared/Translate/RU/UI_RU.txt (hunk 1 above)
Comment (MRSEEO) [reply Contents/mods/zomboid-cnd-speech/media/lua/shared/Translate/RU/UI_RU.txt:328]: No. It also works in Russian

Comment (ChuckTheSheep) [reply Contents/mods/zomboid-cnd-speech/media/lua/shared/Translate/RU/UI_RU.txt:328]: If any of these words don't have a translation you can actually take them out of the list- the code will still work. Just make sure the numbers at the end of the ID are in order (1,2, 3, etc). Again, great job!

**Review by ChuckTheSheep — COMMENTED** [review]
Great job, I'll merge this later today.

**Comment by ChuckTheSheep** [comment]
Does this work for you in game? I can not seem to get the encoding to work properly in game.

**Comment by MRSEEO** [comment]
> Does this work for you in game? I can not seem to get the encoding to work properly in game.

Tested the new update from workshop. Yeah, it works pretty well!

**Comment by ChuckTheSheep** [comment]
Yes, the encoding was giving me some trouble at first. Thanks for the hard work!

**Pull request merged**